samples/graph-teams-tag/csharp/GraphTeamsTag/ClientApp/src/components/view-or-edit-tag.jsx

```diff
@@ -0,0 +1,282 @@
+// <copyright file="view-or-edittag.tsx" company="Microsoft Corporation">
```

Comment:
we can rename this file as view-edit-tag.jsx

Reply:
Done